examples/painter/app/views/row.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react'
+import withIntent from '../../../../src/addons/with-intent'
 
-function Cell ({ x, y, active, onClick }) {
+const Cell = function ({ x, y, active, onClick }) {
```

Comment:
Ack. Bad flow (and a good catch). I wrapped `Cell` in `withIntent`, but then found it cleaner just to do it in `Row`. I'll revert this.

Thank you!